### PR TITLE
Fix for batchnorm attribute `axis` list case by adding transpose layer

### PR DESCRIPTION
**Pull request by vinitra**
Update batch norm list case as done in PR #117 
Add cosmetic changes to fit into code base

"Fix issue with BatchNormalisation layer having a list axis not adding the required Transpose layer when converting a NHWC model." - @fmannhardt

### CODE REVIEW

File: keras2onnx/ke2onnx/batch_norm.py [hunk 1]
```diff
@@ -17,6 +17,14 @@ def convert_keras_batch_normalization(scope, operator, container):
     else:
         raise AttributeError('There is no input_shape or _input_shape for the operator: ' + operator.full_name)
 
+    if isinstance(op.axis, list):
+            if len(op.axis) == 1:
```
Comment (jiafatom) [inline keras2onnx/ke2onnx/batch_norm.py]: indent issue?

File: keras2onnx/ke2onnx/batch_norm.py [hunk 2]
```diff
@@ -17,6 +17,14 @@ def convert_keras_batch_normalization(scope, operator, container):
...
+                axis = op.axis[0]
+            else:
+                raise AttributeError('No support for more than one axis in: ' + operator.full_name)
+        else:
```
Comment (jiafatom) [inline keras2onnx/ke2onnx/batch_norm.py]: indent issue?

File: keras2onnx/ke2onnx/batch_norm.py (hunk 1 above)
Comment (vinitra) [reply keras2onnx/ke2onnx/batch_norm.py]: fixed!

File: keras2onnx/ke2onnx/batch_norm.py (hunk 2 above)
Comment (vinitra) [reply keras2onnx/ke2onnx/batch_norm.py]: fixed!
